pages/common/octave.md

```diff
@@ -0,0 +1,28 @@
+# octave
+
+> GNU Octave is a programming language for scientific computing..
+> More information: <https://docs.octave.org/latest/Invoking-Octave-from-the-Command-Line.html>.
+
+- Start an interactive Octave session:
+
+`octave`
+
+- Execute a script file:
+
+`octave {{path/to/script.m}}`
+
+- Execute a script file and pass arguments:
```

Comment:
```suggestion
- Execute a script file with arguments:
```

I'm not a native english speaker, imho it sounds better this way, what do you think ?

Reply:
> I'm not a native english speaker, imho it sounds better this way, what do you think ?

This looks better to me.